torch_xla/core/xla_model.py

```diff
@@ -37,9 +37,14 @@
 XLA_LIB = Library("xla", "DEF")
 
 from . import xla_model as this_module
-xrt_world_size = deprecated(this_module, torch_xla.runtime.world_size)
-get_ordinal = deprecated(this_module, torch_xla.runtime.global_ordinal)
-parse_xla_device = deprecated(this_module, _utils.parse_xla_device)
+xrt_world_size = deprecated(this_module, torch_xla.runtime.world_size,
+                            'xrt_world_size() will be removed in release 2.6.')
```

Comment:
a few comments to improve our deprecation approach:

+ Do we have an existing set of action issues defined for future API removals and maintain that list on github to tackle when the time comes
+ Can the message suggest what to replace the API with?
+ Please confirm the API transition is safe and bug free by adding tests that compare old API and new API functionality
+ This message hard codes the version number - I expect the version number can vary from one API to another - can we have the version be parameterized for easier maintenance?
+ We need a RFC that outlines our backward guarantees 

cc @JackCaoG @jiawenliu64 @zpcore @qihqi

Reply:
What about using this table (https://github.com/pytorch/xla/issues/7739) for tracking?  We can add a column for deprecate timeline.